packages/odyssey-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -50,28 +52,99 @@ export const Tooltip = withTheme(
   forwardRef<HTMLElement, TooltipProps>((props, ref) => {
     const { children, id, label, position = "top", ...rest } = props;
 
+    const tooltipRef = useRef<HTMLDivElement>(null);
     const oid = useOid(id);
     const omitProps = useOmit(rest);
     const tooltipClasses = useCx(styles.root, styles[`${position}Position`]);
     const clone = cloneElement(children, { "aria-describedby": oid });
+    const [coords, setCoords] = useState({});
```

Comment:
this state might be better named as `tooltipStyles` or similar to reflect the fact it captures visibility/opacity and other styles